docker/buildkite/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk/openjdk11:alpine
+FROM adoptopenjdk/openjdk11@sha256:13475f4be7d6701a2ebd0516c4c929f2fa78ea773b4483bb580a36a04975e9f9
```

Comment:
Nit: since people like me don’t know the context, would you mind leaving some comments about why we have to pin the sha revision ?